src/olympia/addons/views.py

```diff
@@ -822,3 +823,16 @@ class ReplacementAddonView(ListAPIView):
     authentication_classes = []
     queryset = ReplacementAddon.objects.all()
     serializer_class = ReplacementAddonSerializer
+
+
+class CompatOverrideView(ListAPIView):
+    queryset = CompatOverride.objects.all()
+    serializer_class = CompatOverrideSerializer
+
+    def filter_queryset(self, queryset):
+        # Use the same Filter we use for AddonSearchView for consistency.
+        guid_filter = AddonGuidQueryParam(self.request)
+        guids = guid_filter.get_value()
```

Comment:
What should we do when no guid is passed? Return a 400? This should be tested.

Reply:
Hmm.  Not sure - currently you'd just get no results, as you would if didn't specify the `guid` parameter.  But I guess throwing an error is clearer when the endpoint only has one function.

Reply:
Avoids a useless db query with `guid=''` as well :)

Reply:
https://github.com/mozilla/addons-server/pull/7339/commits/39781e9698733b84f8239ad2ac6232f9c64679ae